CONTRIBUTING.asciidoc

```diff
@@ -49,89 +49,98 @@ The major technology stacks have their own repository carrying the prefix `devon
 Tools we provide typically have a repository named like the tool they provide (omitting prefixes like `devon[fw]`):
 
 * https://github.com/devonfw/tools-cobigen[CobiGen] (incremental code generator)
-* https://github.com/devonfw/ide[devon-ide] (development environment)
+* https://github.com/devonfw/ide[devonfw IDE] (development environment)
 * https://github.com/devonfw/sonar-devon4j-plugin[sonar-devon4j-plugin] (SonarQube plugin for architecture validation)
 * https://github.com/devonfw/cicdgen[CICDgen] (generator for CI/CD)
-* https://github.com/devonfw/devon-docgen[docgen] (generator for AsciiDoc to PDF and other formats)
-* https://github.com/devonfw-forge/solicitor[solicitor] (License compatibility checker)
+* https://github.com/devonfw/devon-docgen[DocGen] (generator for AsciiDoc to PDF and other formats)
```

Comment:
https://github.com/devonfw/docgen[DocGen]

Reply:
GitHub seems to mangle this link, but in AsciiDoc it should work correctly.
Should this be called "devonfw-docgen" or something else, or ist "DocGen" fine?